app/services/chat_completion_api.rb

```diff
@@ -127,4 +127,4 @@ def self.model_token_limit(name)
       'gpt-3.5-turbo-instruct' => 4096
     }[name]
   end
-end
```

Comment:
My editor likes to add these newlines at the end of files. If that's problematic, I'll be sure to disable it and can revert this change.

Reply:
Do you mean your editor removes these newlines? If I'm reading this diff correctly it looks there was a newline after the "end" and now it's removed it?

It's fine, but I would love to standardize whitespacing handling somehow. Do you know if there a dotfile I can add to the project that will cause all IDEs to handle whitespace the same?

Reply:
I’ve had success with https://editorconfig.org/

Reply:
I’ve used https://prettier.io/ as well, but find it most useful for languages other than ruby.

Reply:
Done. Thanks. I just did a commit in main that adds editorconfig and added it to the README